src/routes/+page.svelte

```diff
@@ -23,8 +23,12 @@
 		el.scrollIntoView({
 			behavior: 'smooth'
 		});
-		// focus the input in the given section once in view
-		document.getElementById(key)?.focus();
+
+		// Focus the input in the given section once in view, preventing duplicate scrolling.
```

Comment:
Great comments! 

I initially added the focus so that a keyboard-only user would be able to just adjust the slider once they are scrolled to the new section, but with these changes they have to press tab (or Option+Tab in Safari) to highlight it. I think this is a fine experience too, so really this behavior is for VoiceOver users, and IMO works nicely for that!

Reply:
Gotcha, that makes sense. Thanks Chelsea! :)